File: VB/ConditionalAppearanceExample.Module.Win/Controllers/WinConditionalAppearanceController.vb
Imports System
Imports System.ComponentModel
Imports System.Collections.Generic
Imports System.Diagnostics
Imports System.Text

Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Actions
Imports DevExpress.Persistent.Base

Imports DevExpress.ExpressApp.ConditionalAppearance
Imports ConditionalAppearanceExample.Module.Controllers
Imports DevExpress.ExpressApp.Win.Editors
Imports DevExpress.XtraEditors.Controls
Imports System.Drawing

Namespace ConditionalAppearanceExample.Module.Win.Controllers
   Partial Public Class WinConditionalAppearanceController
       Inherits ConditionalAppearanceController

      Protected Overrides Sub CustomizeDisabledEditorsAppearance(ByVal e As ApplyAppearanceEventArgs)
         MyBase.CustomizeDisabledEditorsAppearance(e)
         Dim dxEditor As DXPropertyEditor = TryCast(e.Item, DXPropertyEditor)
         If dxEditor IsNot Nothing AndAlso dxEditor.Control IsNot Nothing Then
            dxEditor.Control.Properties.Appearance.BackColor = Color.RosyBrown
            dxEditor.Control.Properties.BorderStyle = BorderStyles.Simple
         End If
      End Sub
   End Class
End Namespace

File: CS/ConditionalAppearanceExample.Module.Win/Controllers/WinConditionalAppearanceController.cs
using System;
using System.ComponentModel;
using System.Collections.Generic;
using System.Diagnostics;
using System.Text;

using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Actions;
using DevExpress.Persistent.Base;

using DevExpress.ExpressApp.ConditionalAppearance;
using ConditionalAppearanceExample.Module.Controllers;
using DevExpress.ExpressApp.Win.Editors;
using DevExpress.XtraEditors.Controls;
using System.Drawing;

namespace ConditionalAppearanceExample.Module.Win.Controllers {
   public partial class WinConditionalAppearanceController : ConditionalAppearanceController {

      protected override void CustomizeDisabledEditorsAppearance(ApplyAppearanceEventArgs e) {
         base.CustomizeDisabledEditorsAppearance(e);
         DXPropertyEditor dxEditor = e.Item as DXPropertyEditor;
         if (dxEditor != null && dxEditor.Control != null) {
            dxEditor.Control.Properties.Appearance.BackColor = Color.RosyBrown;
            dxEditor.Control.Properties.BorderStyle = BorderStyles.Simple;            
         }
      }
   }
}
